nautobot/core/apps/__init__.py

```diff
@@ -50,6 +69,9 @@ def ready(self):
         if menu_items is not None:
             register_menu_items(menu_items)
 
+        jobs = import_string("nautobot.core.jobs.jobs")
+        register_jobs(jobs)
```

Comment:
This is where `foo_jobs_module.jobs` (the attribute that contains a list of Job classes) is load and could be the point where we align registration for both Nautobot and Celery simultaneously.